Tests/configure_and_test_integration_instances.py

```diff
@@ -302,7 +310,9 @@ def set_integration_params(integrations, secret_params, instance_names):
         (bool): True if integrations params were filled with secret configuration values, otherwise false
     """
     for integration in integrations:
-        integration_params = [item for item in secret_params if item['name'] == integration['name']]
+        changes_map = {'%%SERVER_HOST%%': server_host}
```

Comment:
I see you added the param server_host and this is the only use for this in this integration (same is true in the `set_integration_params` method). 

If the idea is that the change_map can grow in the future with additional mappings, it might be wiser to create the change_map in the main function and send it instead of the server_host specifically - I think it would be easier to expand if needed.